client/hotrod-client/src/main/java/org/infinispan/client/hotrod/RemoteCacheManagerAdmin.java

```diff
@@ -24,6 +24,16 @@
    @Override
    <K, V> RemoteCache<K, V> createCache(String name, String template) throws HotRodClientException;
 
+   /**
+    * Creates a cache on the remote server cluster using the specified template.
+    *
+    * @param name the name of the cache to create
+    * @param template {@link DefaultTemplate} enum
+    * @return the cache
+    * @throws HotRodClientException
+    */
+   <K, V> RemoteCache<K, V> createCache(String name, DefaultTemplate template) throws HotRodClientException;
```

Comment:
Yes because as a user I might not know where to pick up this String from, but the DefaultTemplate is strongly typed so I discover it easily and it's nice to be used in the examples